pom.xml

```diff
@@ -8,13 +8,18 @@
     <dependency>
       <groupId>io.pravega</groupId>
       <artifactId>pravega-client</artifactId>
-      <version>0.1.0-SNAPSHOT</version>
+      <version>0.3.0-64.d0c8497-SNAPSHOT</version>
```

Comment:
Ironically we are striving for the opposite, to stabilize the builds by using a specific version.   What problem are you worried about?

Reply:
Given that this project depends on very few classes on the pravega client (SteamManager, ClientConfig, ClientFactory) and it is a much simpler case than, e.g., flink-connectors, I believe it makes more sense to follow the latest build closely, and to avoid "oh crap moment" that we forgot to update it, especially when this project is rarely attended.  We can enable Travis integration for this project and pick up the broken changes as earlier as possible.

Reply:
We'll agree to disagree maybe but I am not supportive of weak versioning.